punjabi/punjabi2.md

```diff
@@ -0,0 +1 @@
+Click [here](https://en.wikipedia.org/wiki/List_of_Punjabi_people) to know about famous Punjabi people.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.